pygotham/frontend/about.py

```diff
@@ -1,28 +1,35 @@
 """About PyGotham."""
 
-from flask import Blueprint
+from collections import defaultdict
 
-from pygotham.frontend import direct_to_template
+from flask import Blueprint, render_template, url_for
+
+from pygotham.events import get_current
+from pygotham.models import AboutPage
 
 __all__ = ('blueprint',)
 
 blueprint = Blueprint('about', __name__, url_prefix='/about')
 
-direct_to_template(
-    blueprint,
-    '/code-of-conduct',
-    template='about/code-of-conduct.html',
-    endpoint='code_of_conduct',
-)
-direct_to_template(
-    blueprint,
-    '/privacy-policy',
-    template='about/privacy-policy.html',
-    endpoint='privacy_policy',
-)
-direct_to_template(
-    blueprint,
-    '/faq',
-    template='about/faq.html',
-    endpoint='faq',
-)
+
+def get_nav_links():
+    """Generates all about page titles and urls for use in the navbar."""
+    links = defaultdict(dict)
```

Comment:
Should this be `links = defaultdict(OrderedDict)`? Would `link` need to be an `OrderedDict` as well?